src/ProofOfConcepts/CommunicationError/CommunicationErrorStatusNoSubscripton.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import {
+  Alert,
+  AlertActionCloseButton,
+  ClipboardCopy,
+} from '@patternfly/react-core';
+
+import { useTranslation } from '@rhoas/app-services-ui-components';
+
+import './CommunicationErrorStatus.css';
+
+export const ExpandableAlerts = () => {
+  const { t } = useTranslation('cos-ui');
+
+  return (
+    <React.Fragment>
+      <Alert
+        isExpandable
+        isInline
+        variant="danger"
+        title={t('communicationErrorTitle')}
+        actionClose={
+          <AlertActionCloseButton
+            onClose={() => alert('Clicked the close button')}
+          />
+        }
+        actionLinks={<React.Fragment></React.Fragment>}
+      >
+        <p>{t('communicationErrorDescripton')}</p>
+        <p>
+          <br />
+        </p>
+        <p>
+          {t('communicationErrorMessageNoSubscripton')}
+          <ClipboardCopy
+            className="pf-c-clipboard-copy__text-pf-m-code"
+            hoverTip="Copy"
+            clickTip="Copied"
+            variant="inline-compact"
+            isCode
+          >
+            {t('communicationErrorEmail')}
```

Comment:
@gashcrumb When I chatted with @Kells512 about implementing this part, I mentioned that the behavior we discussed during design review isn't actually supported with the current clipboard copy component. I raised [a question in PF slack](https://patternfly.slack.com/archives/C4FM977N0/p1670509828573799) about what would be recommended for this situation, but there were no replies. 

The behavior we had discussed during review was that the email address that displays as a link should be implemented using the `href="mailto:..."` method so that the user can click the email address to open their email client, and the copy icon would be a separate button. 

What would you suggest? I have no idea how hard it is to recreate the copy functionality that we get for free with the clipboard copy component. One option is to leave it as it is here, which means the email address only gets copied when the text is clicked. And later figure out if we need to open an issue in PF or implement this copy behavior ourselves.

Reply:
Implementing a custom element for this should not be tough, but I guess we can close this PR as it is and have it implemented as an improvement subsequently.